plugin/plugin.go

```diff
@@ -99,13 +99,14 @@ func (p *Plugin) Find(ctx context.Context, droneRequest *config.Request) (*drone
 	}
 
 	// cleanup
-	configData = strings.ReplaceAll(configData, "...", "")
+	configData = string(removeDocEndRegex.ReplaceAllString(configData, ""))
 	configData = string(dedupRegex.ReplaceAll([]byte(configData), []byte("---")))
 
 	return &drone.Config{Data: configData}, nil
 }
 
 var dedupRegex = regexp.MustCompile(`(?ms)(---[\s]*){2,}`)
+var removeDocEndRegex = regexp.MustCompile(`(?ms)(^[\s]*\.\.\.[\s]*)+$`)
```

Comment:
Are spaces around `...` valid als yaml? If not we could simply match for `^\.\.\.$

This regex could break with something like this:
```yaml
data: |
  this is
  ...
  some data
```

Did you notice any situations where the leading spaces produced by the concatenation?

Reply:
good point! fixing this now

Reply:
@foosinn PR updated.